### DISCUSSION TITLE
Preview nuget for SkiaSharp 3

### DISCUSSION THREAD

**Discussion by taublast** [wieslawsoltes/Svg.Skia#245] [Ideas]
Hi @wieslawsoltes, please can we get a preview nuget targeting skiasharp v3 preview?

Currently just using a custom build ("Svg.SkiaSharp3" preview nuget) of your awesome lib to be able to use it along with v3, also I know that MapsUi (https://github.com/Mapsui/Mapsui/discussions/2763) cannot have their v3-targeting preview nuget released as they rely on Svg.Skia too, a bit shame to just wait until release..

Many thanks for your work!

**Comment by wieslawsoltes** [comment]
Why you need that? Just reference 3.0 in your project.

**Comment by taublast** [comment]
Indeed, i was working this around from 3.0.0-preview.2.1 when some API were removed in v3.. now restored for backward compatibility. Thanks! :)
